engine/rpcserver.go

```diff
@@ -604,7 +604,7 @@ func createAccountInfoRequest(h account.Holdings) (*gctrpc.GetAccountInfoRespons
 	accounts := make([]*gctrpc.Account, len(h.Accounts))
 	for x := range h.Accounts {
 		var a gctrpc.Account
-		a.Id = h.Accounts[x].ID
+		a.Id = h.Accounts[x].Credentials.String()
```

Comment:
I appreciate this add. I do want to highlight something though. Take the following command:
```bash
go run . --apikey="main" --apisecret="main" --apisubaccount="main" --timeout=5m getaccountinfo ftx spot
```
and the response:
```bash
{
 "exchange": "ftx",
 "accounts": [
  {
   "id": "Key:[sub...] SubAccount:[sub] ClientID:[]",
   "currencies": [
    {
     "currency": "USD",
     "total_value": 1337,
     "hold": 1337,
     "free": 1337,
     "free_without_borrow": 1337,
    },
    {
     "currency": "BTC",
     "total_value": 1337,
     "free": 1337,
     "free_without_borrow": 1337,
    }
   ]
  }
 ]
}
```
What I'm seeing here is that the response results and credentials are for my config's credentials, and not the supplied ones.

However, the command `UpdateAccountInfo` behaves how I would expect. Since running that, I get the following calling `GetAccountInfo`:
```bash
{
 "exchange": "ftx",
 "accounts": [
  {
   "id": "Key:[sub...] SubAccount:[sub] ClientID:[]",
   "currencies": [
    {
     "currency": "USD",
     "total_value": 1337,
     "hold": 1337,
     "free": 1337,
     "free_without_borrow": 1337,
    },
    {
     "currency": "BTC",
     "total_value": 1337,
     "free": 1337,
     "free_without_borrow":1337,
    }
   ]
  },
  {
   "id": "Key:[main...] SubAccount:[main] ClientID:[]",
   "currencies": [
    {
     "currency": "CEL",
     "free": 1337,
    },
    {
     "currency": "AUD",
     "free": 1337,
    },
    {
     "currency": "BTC",
     "total_value":1337,
     "free": 1337,
     "free_without_borrow": 1337,
    },
    {
     "currency": "ETH",
     "total_value": 1337,
     "free": 1337,
     "free_without_borrow": 1337,
    },
   ...
   ]
  },
  {
   "id": "Key:[main...] SubAccount:[main] ClientID:[]",
   "currencies": [
    {
     "currency": "USD",
     "total_value": 1337,
     "hold": 1337,
     "free": 1337,
     "free_without_borrow":1337,
    },
    {
     "currency": "BTC",
     "total_value": 1337,
     "free":1337,
     "free_without_borrow": 1337,
    }
   ]
  }
 ]
}
```

So I'm requesting that for the command `GetAccountInfo` returns an error on credentials being supplied, as it actually cannot process that. You could limit the response as another idea

Reply:
Double fun, that last `GetAccountInfo` in the above output should be of subaccount `sub` with `main` credentials.

It renders it properly in `UpdateAccountInfo`